sdk/eventhub/eventhubs-checkpoint-blob/src/log.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import debugModule from "debug";
+/**
+ * @ignore
+ * log statements for error
+ */
+export const error = debugModule("azure:event-hubs:error");
```

Comment:
Should this be `azure:eventhubs-checkpoint-blob`?

Reply:
Nice catch! Thank you :)